src/components/VFourOhFour.vue

```diff
@@ -71,7 +71,6 @@ export default defineComponent({
     VLink,
     VSearchBar,
   },
-  props: ['error'],
```

Comment:
Strangely I get an error here for all the `defineComponent` block: "No overload matches this call." but is present in the main branch too. Can we inspect this file and add it to the `tsconfig.json`?

Also, the script section is missing `lang="ts"`.

Reply:
This error is caused by the `.svg?inline` components that come with `@nuxtjs/svg` module. I will open an issue to replace them with `<svg>` element because that module seems to be incompatible with TS, and it is not supported in Nuxt 3.

Reply:
I see, thanks for clarifying the error cause 👍